pontoon/administration/forms.py

```diff
@@ -25,6 +26,16 @@ class ProjectForm(forms.ModelForm):
     locales_readonly = forms.ModelMultipleChoiceField(
         queryset=Locale.objects.all(), required=False,
     )
+    locales = forms.ModelMultipleChoiceField(
+        queryset=Locale.objects.all(), required=False,
+    )
+
+    def clean(self):
+        cleaned_data = super().clean()
+        locales_readonly = cleaned_data.get("locales_readonly")
+        locales = cleaned_data.get("locales")
+        if not (locales or locales_readonly):
+            raise ValidationError("This field is required")
```

Comment:
The message should read "At least one locale must be selected.", otherwise the UI implies the localizable locale field is required.